starter_code/express-spotify-app/views/artists.ejs

```diff
@@ -0,0 +1,14 @@
+
+<section>
+
+    <% artists.forEach((artist) => { %>
+        <div>
+            <h1>Your search result:</h1>
+            <h2><%= artist.name %></h2>
+            <% if (artist.images.length > 0) { %>
```

Comment:
Good call! However, the best approach is define a helper function that, if there is an image, return it, otherwise displays a placeholder. Then you can use that function in albums.ejs too.